packages/wallets/src/components/Base/Divider/Divider.tsx

```diff
@@ -8,7 +8,7 @@ type TProps = {
 };
 
 const Divider: React.FC<TProps> = ({ color = '#f2f3f4', margin, variant = 'horizontal' }) => (
```

Comment:
for future prs/refactors, as I know it was introduced before this PR - we shouldn't be managing css from javascript, as it defeats the whole purpose of .css files and its very much agains separation of concern,